### PR TITLE
Implement song list page and search page with placeholder data.

### PR DESCRIPTION
**Pull request by Aleks-Ada**
Implements part of #3.

### CODE REVIEW

**Comment by Aleks-Ada** [comment]
Any review comments, @AlexM1tra?

**Comment by AlexM1tra** [comment]
I think it would be good to create smaller pull requests in future but looks good to me.

File: aldify/src/requests.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import {Song} from "@/models/song";
+
+export function fetchAllSongs() : Promise<Song[]> {
+    return fetchSongs("*");
+}
+
+export function fetchSongs(query: string): Promise<Song[]> {
```
Comment (AlexM1tra) [inline aldify/src/requests.ts:7]: nit: let's file an issue to swap out the mock backends with the real ones.

**Review by AlexM1tra — APPROVED** [review]
LGTM, Approved

File: aldify/src/requests.ts (hunk 1 above)
Comment (Aleks-Ada) [reply aldify/src/requests.ts:7]: Sure, thanks for mentioning.

Comment (Aleks-Ada) [reply aldify/src/requests.ts:7]: Created issue #5.

**Pull request merged**